rskj-core/src/main/java/co/rsk/peg/utils/BridgeEventLoggerImpl.java

```diff
@@ -258,6 +258,16 @@ public void logReleaseBtcRequestRejected(String sender, Coin amount, RejectedPeg
         this.logs.add(new LogInfo(BRIDGE_CONTRACT_ADDRESS, encodedTopics, encodedData));
     }
 
+    @Override
+    public void logBatchPegoutCreated(BtcTransaction btcTx, byte[] rskTxHash) {
+        CallTransaction.Function event = BridgeEvents.BATCH_PEGOUT_CREATED.getEvent();
+        byte[][] encodedTopicsInBytes = event.encodeEventTopics(rskTxHash);
```

Comment:
The indexed value here is the btc tx hash, that should be the topic

Reply:
Resolved